statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -1287,17 +1461,20 @@ def reorder(self, order):
 #-------------------------------------------------------------------------------
 # VAR Diagnostics: Granger-causality, whiteness of residuals, normality, etc.
 
-    def test_causality(self, equation, variables, kind='f', signif=0.05,
-                       verbose=True):
-        """Compute test statistic for null hypothesis of Granger-noncausality,
-        general function to test joint Granger-causality of multiple variables
+    def test_causality(self, causing, kind='f', signif=0.05, verbose=True):
```

Comment:
to answer my question: `equation` is the equation of the VAR, i.e. the dependent variable, or `caused` in this context. `variable` is then the `causing` variable. (i.e. wald/F test that `variable` in `equation` is insignificant.)

I don't think there is a way to switch the positional arguments in a backwards compatible way.